test/indexer/data/aaa96233-bf27-44c7-82df-b4dc15ad4d9d.2018-11-02T11:33:44.698028Z.results.json

```diff
@@ -1,4 +1,80 @@
 [
+    {
+        "_id": "bundles_aaa96233-bf27-44c7-82df-b4dc15ad4d9d_50b88c774d130ce5cb9e680241549c2df665efca",
+        "_index": "azul_v2_nadove_test_replica",
```

Comment:
Wrong/inconsistent index name. Why doesn't the test catch that?

Reply:
Because the index name is replaced to match the current deployment when the file is loaded. https://github.com/DataBiosphere/azul/blob/3e6370d8500db776f705d4e6e197ce70f39a5c25/test/indexer/__init__.py#L212